testnet/stacks-node/src/neon_node.rs

```diff
@@ -419,6 +419,16 @@ fn try_mine_microblock(
                                 // will need to relay this
                                 next_microblock = Some(microblock);
                             }
+                            Err(NetError::ChainstateError(err_str)) => {
```

Comment:
If not done in this PR, https://github.com/blockstack/stacks-blockchain/pull/2234 will turn the `NetError` returned by this function into a `ChainStateError` so we don't have to match against a string. For some reason, all the codepaths are returning ChainStateErrors, but they are being wrapped into a net_error

Reply:
Ah -- yeah, that's better. I'm going to leave this check in this PR, and we can switch to matching with the ChainstateError enum in #2234 later on, because I'm irrationally annoyed by this warn :)